web/components/feed/feed-timeline-items.tsx

```diff
@@ -201,6 +146,25 @@ export const FeedTimelineItems = (props: {
   )
 }
 
+export function FeedRelatedItemFrame(props: {
+  children: React.ReactNode
+  href: string
+  className?: string
+}) {
+  const { children, href, className } = props
```

Comment:
This used to have a `useIsVisible` hook to market feed items as seen, which is/will be important for feed functionality. I can't find it moved anywhere else so this should be re-added here.

Reply:
I think this a different thing? FeedItemFrame is still intact, I've just added FeedRelatedItemFrame
![Screenshot 2023-06-14 at 3 49 04 PM](https://github.com/manifoldmarkets/manifold/assets/46611122/953483d6-7ea4-49f5-8d48-546c9132b8ed)

Reply:
@IanPhilips

Reply:
oh great, if you didn't change that then no worries!